common/org.testeditor.dsl.common.testing/src/org/testeditor/dsl/common/testing/ResourceSetHelper.xtend

```diff
@@ -0,0 +1,53 @@
+package org.testeditor.dsl.common.testing
+
+import com.google.inject.Provider
+import javax.inject.Inject
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.resource.XtextResourceSet
+import org.testeditor.aml.AmlModel
+import org.testeditor.tcl.TclModel
+import org.testeditor.tsl.TslModel
+import org.eclipse.xtend.lib.annotations.Accessors
+
+@Accessors
+class ResourceSetHelper {
```

Comment:
Also I don't want to call `setUpResourceSet` all the time. This can be easily solved:

```
@Inject
new(XtextResourceSet resourceSet) {
   // do some initialization here
}
```